### PR TITLE
[Content] Quarg Acknowledges Player Flying An Arfecta And Warns Them

### PR DESCRIPTION
**Pull request by rovermicrover**
## Summary
Adds an event on Quarg stations where if the player flagship is an Arfecta the pug warn the player of the ships power.

Adds some world building, and some more uniqueness to the Arfecta.

Fiction writing is outside my normal wheel house but trying to work on some new skills. 😊 Let me know if the dialogue meets the feel of a Quarg. 

## Save File
This save file can be used to play through the new mission content:
[Save File](https://www.dropbox.com/s/iz2clkddmk6yhox/Rover%20Freewing~~previous-1.txt?dl=0)

### CODE REVIEW

File: data/quarg/quarg missions.txt [hunk 1]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "station"
```
Comment (RisingLeaf) [inline data/quarg/quarg missions.txt:272]: why only on stations?

Comment (rovermicrover) [reply data/quarg/quarg missions.txt:272]: At first I just copied the "Ask Quarg About Coalition Early" mission to get started.

I looked at that  source condition and thought maybe I should remove it, but then I thought there was more narrative weight if they warned you about the power of the ship, a ship that was most likely used to destroy one of their rings, on one of their rings.

File: data/quarg/quarg missions.txt [hunk 2]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+	on offer
+		conversation
+			`As soon as you step off your ship a rather imposing Quarg blocks your exit ramp while stairing piercingly at your ship.`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:275]:
```suggestion
			`As soon as you step off your ship a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
```

No need to impose a ramp on the Arfecta here where it's not actually relevant.

File: data/quarg/quarg missions.txt [hunk 3]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgusted.`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`"It is as though a baby in swaddling has killed a dragon, and taken its treasure for its own..." it says with a mix of amusement and disgust.`
```

Comment (EjoThims) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`"It is as though a baby in swaddling has killed a dragon, and now rides its corpse, thinking to have mastered fire..." it says with a mix of amusement and disgust.`
```
A more fitting analogy, imo, for both the situation and the ship. You haven't looted something the dragon guards, you have desecrated and weapon used the dragon's own body.

File: data/quarg/quarg missions.txt [hunk 4]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
+				goto end
+			label righteous
+			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"And you believe yourself in a position to judge?" the Quarg says irritated.`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:293]:
```suggestion
			`	"And you believe yourself in a position to judge?" the Quarg says irritated.`
```

This really doesn't follow from the lead in to me. Not even just thematically, no judgement was being passed.

I'd suggest something more directly calling out the arrogant hypocrisy in the line this one is responding to.

File: data/quarg/quarg missions.txt [hunk 5]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
+				goto end
+			label righteous
+			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:290]:
```suggestion
			`	"Did not your Jesus compare himself to this David also?" the Quarg says with what you assume is a sarcastic tone.`
```
This one hits me squarely wrong in many ways.

It demonstrates the Quarg have a specific understanding of a single religion that runs deeper than their grasp of the language as a whole, but also implies that they understand -only- such one specific religion; they are responding as one believer to another, not as an outsider. Even then, the phrasing here, more than the others, implies the player is a believer as well, which is entirely inappropriate.

This would be much better either as a response with a parable from a differing religion (the more different, the better) or simply outright ignorance of the reference.

Would be best served, imo, with replacing the initial reference entirely -in addition- to the above.

File: data/quarg/quarg missions.txt [hunk 6]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+				goto end
+			label anarchy
+			`	"And you believe yourself in a position to judge?" the Quarg says irritated.`
+				goto end
+			label greedy
+			`	"A baby in swaddling as I said before."`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:296]:
```suggestion
			`	"A baby in swaddling, as I said before."`
```

File: data/quarg/quarg missions.txt [hunk 7]
```diff
@@ -259,3 +259,44 @@ mission "Ask Quarg About Coalition Early"
...
+				goto end
+			label end
+			`	"We give you a warning. You know not what you have child, the weapon you wield."`
+			`	"We have seen its power. We have felt its power. Be careful that you are this ship's captain, not its thrall."`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:300]:
```suggestion
			`	"We have seen its power. We have felt its power. Be careful that you are this ship's captain, not its thrall."`
```

This could do with a major tone shift away from the ship/weapon itself being such a threat and towards chastising the impetuous and ignorant nature of the acquisition.

**Review by EjoThims — CHANGES_REQUESTED** [review]
Grammar is mostly good, but a few slips.

Biggest issue is the limited scope of the heavy reliance on religious imagery, especially in the Quarg's ease of understanding it.

But even setting that aside, I'm not sure the overall implication of this specific ship and weapon being so devastating and feared by the Quarg is the direction this should take. After seeing this, I'm left with the implication that the ring was destroyed by a few Arfectas focusing their gridfires, and that's simply absurd in and of itself, as well as limiting further upwards escalation of Pug weaponry.

I do really like the idea of having this acknowledgment happening with an in game confrontation like this (and would even like to see a second that crops up if you're riding around with all three after having had this one), I just think this misses the mark, in several different ways.

File: data/quarg/quarg missions.txt (hunk 5 above)
Comment (Hurleveur) [reply data/quarg/quarg missions.txt:290]: I gotta agree with that, lets avoid references to actual earth religions.
Myths are fine though

**Comment by Arachi-Lover** [comment]
The Quarg would've learned much from the ringworld break, and would've been forced to pick up Warfare 101 for the first time ever, so given how long it's been since that incident, I'd certainly find it very weird if they fear Arfectas as seems implied here, rather than loathing it, but knowing they have developed more ships and outfits that can better fight the Pug T3 ones.

I think it's best to focus on the Quarg being visibly irritated at the Pug ship being there, and at you as well for bringing it there (even if technically you did a good for their side in killing some Pug and stealing an Arfecta from them). You could also have it so that while there's this one Quarg talking to you, there are also hundreds more around, in a crowd that's awfully agitated by Quarg standards.

**Comment by rovermicrover** [comment]
> The Quarg would've learned much from the ringworld break, and would've been forced to pick up Warfare 101 for the first time ever, so given how long it's been since that incident, I'd certainly find it very weird if they fear Arfectas as seems implied here, rather than loathing it, but knowing they have developed more ships and outfits that can better fight the Pug T3 ones.

I was trying to have the Quarg been only slightly irritated at the ship itself. I was trying to imply that the Quarg wasn't afraid of the ship, but rather a human piloting it, and what that meant for humanaity. Similar to their attitude around the Emerald Sword and their warning their. I can add some text around the warning here to make that clearer.

> I think it's best to focus on the Quarg being visibly irritated at the Pug ship being there, and at you as well for bringing it there (even if technically you did a good for their side in killing some Pug and stealing an Arfecta from them). You could also have it so that while there's this one Quarg talking to you, there are also hundreds more around, in a crowd that's awfully agitated by Quarg standards.

I actually had it written that way, and then switched to just the one Quarg, because I was under the impression that the Quarg were either a have mind, or had some type of shared memory or experience. So there wouldn't be a need for them to congregate, they would all already know of the ships landing and presence through the one Quarg's experience.

My impression could be off here though.

File: data/quarg/quarg missions.txt (hunk 3 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Would changing "killed a dragon" to "tamed a dragon" work too?

So 

"It is as though a baby in swaddling has tamed a dragon, and now rides it through the sky spewing fire with reckless abandon."

File: data/quarg/quarg missions.txt (hunk 7 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:300]: Was going for less the ship being a threat, and more a lesser being in possession of it being a threat to the balance of the galaxy.

Would this help?

"We have seen its power. We have felt its power. We have learned from its power. We have now even overcome its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you are this ship's captain, not its thrall."

File: data/quarg/quarg missions.txt (hunk 1 above)
Comment (Hurleveur) [reply data/quarg/quarg missions.txt:272]: There is a `"ringworld"` attribute for that, which differs from stations

File: data/quarg/quarg missions.txt (hunk 4 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:293]: Would the following work?

"So you took it upon yourself to prove them wrong?" the Quarg says with an amused incredulous smile.

Given that the Quarg may actually agree that the Pug think themselves god, but would still be annoyed with your arrogance.

File: data/quarg/quarg missions.txt (hunk 1 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:272]: Gotcha will make that change. I guess the station in human space doesn't count as a ring world because the ring isn't done yet.

File: data/quarg/quarg missions.txt (hunk 7 above)
Comment (EjoThims) [reply data/quarg/quarg missions.txt:300]: Even with this angle I'd go a bit further with the end.

"Be careful that you direct this power, and not it you."

Could even go so far as to add after that "And remember that we have learned to overcome it."

File: data/quarg/quarg missions.txt (hunk 5 above)
Comment (Hurleveur) [reply data/quarg/quarg missions.txt:290]: I don't know if I want the ES universe to have that religion cannon, especially in that state and with the same names, so yeah I'd prefer a more generic response, especially since not everyone will know about who this is and what they did if we use a specific story.
Also best not to open the can of worm of religious things, some may interpret it badly and whatnot, no need to risk it

**Comment by Hurleveur** [comment]
I don't think the Quargs are a hive mind. Whenever you arrive to them you need to wait before they find the right person for your request. Also given their remark on the Pug they are totally not a hive mind in my opinion actually: 
			`	The Quarg pauses for a moment as if struggling for words. "I am myself, and also I am Quarg. You are yourselves, and also you are human. But we think perhaps a Pug is Pug first, and then also itself."`

File: data/quarg/quarg missions.txt (hunk 7 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:300]: I like that

File: data/quarg/quarg missions.txt (hunk 4 above)
Comment (EjoThims) [reply data/quarg/quarg missions.txt:293]: Yea, that would work very well there I think.

File: data/quarg/quarg missions.txt (hunk 7 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Updated with suggestion

**Comment by rovermicrover** [comment]
> I don't think the Quargs are a hive mind. Whenever you arrive to them you need to wait before they find the right person for your request. Also given their remark on the Pug they are totally not a hive mind in my opinion actually: ` The Quarg pauses for a moment as if struggling for words. "I am myself, and also I am Quarg. You are yourselves, and also you are human. But we think perhaps a Pug is Pug first, and then also itself."`

Will add the crowd back in then.

File: data/quarg/quarg missions.txt [hunk 8]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
+				goto end
+			label righteous
+			`	“You fancy yourself a savior or a prophet then?” the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused incredulous smile.`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:293]:
```suggestion
			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
```

Or even explicitly "and" instead of the comma. Dealer's choice :)

File: data/quarg/quarg missions.txt (hunk 5 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Would a reference to Hercules work? It’s from a religious body of work, just one that is no longer practice and has passed into myth.

Or should I aim for more generic than that?

**Comment by rovermicrover** [comment]
Crowd Added

File: data/quarg/quarg missions.txt (hunk 5 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Updated this path to be more generic let me know if that works.

File: data/quarg/quarg missions.txt [hunk 9]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has tamed a dragon, and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`
```
Comment (EjoThims) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`
```

One more little slip in the rewriting it looks like.

File: data/quarg/quarg missions.txt [hunk 10]
```diff
@@ -273,7 +273,7 @@ mission "Quarg Pug Arfecta Warning"
 	on offer
 		conversation
 			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
-			`"It is as though a baby in swaddling has tamed a dragon, and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`
+			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon." it says with a mix of amusement and disgust.`
```
Comment (roadrunner56) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

**Review by EjoThims — APPROVED** [review]
This all is looking good to me.

File: data/quarg/quarg missions.txt [hunk 11]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`	"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
```
Pretty sure we need to indent everything that is not in the first section of ``

File: data/quarg/quarg missions.txt [hunk 12]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as an act of righteous defiance."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:282]: I love that reference to Nietzsche
Otherwise I'd have said to change it to "I don't believe in Gods"
But since its a ref, keep it!

File: data/quarg/quarg missions.txt [hunk 13]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as an act of righteous defiance."`
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:280]:
```suggestion
				`	"It was an act of righteous defiance."`
```
mb better like this? Caus otherwise it feels like its not smth you say but a description of smth you say to me
Especially considering the Quarg's response

File: data/quarg/quarg missions.txt [hunk 14]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:287]: I think that this line and maybe all others could be missing the usual Quarg elements of visual depictions of their body?
Or at least some more description, like `he scoffs, "it matters not how it happened"`

The Quarg body things happening is valid for all others phrases too I think, the Quarg would not be willing to speak much about it but you should be able to see all kinds of emotions? The same kind we get when we show the broken ring world fragment

File: data/quarg/quarg missions.txt [hunk 15]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
...
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
+				goto end
+			label greedy
+			`	"A baby in swaddling, as I said before."`
+				goto end
+			label end
+			`	"We give you a warning. You know not what you have child, the weapon you wield."`
+			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`
+			`	"And be careful to remember that we have learned to overcome it."`
+			`	The Quarg then curtly motions farewell and leaves you to your business. As you go you feel the crowd's glare follow you.`
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:302]: they motion farewell? would they really do that, in this context?

File: data/quarg/quarg missions.txt [hunk 16]
```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:266]:
```suggestion
	minor
	landing
```

I think this ought to be a minor thing that wont happen in the middle of a story

**Review by Hurleveur — COMMENTED** [review]
Do note I'm not requesting changes, I'm suggesting them. I am no content reviewer, I just have an opinion, like anyone.
I quite that it happens and it seems good how it does, just got comments on how I think it could be made better

File: data/quarg/quarg missions.txt (hunk 15 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:303]: Like a dismissive hand motion from the center down and out. Like see ya and a get out of here motion

File: data/quarg/quarg missions.txt (hunk 14 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:288]: Sounds good. Will make the change

File: data/quarg/quarg missions.txt (hunk 15 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt:303]: Ideas welcome most definitely welcome here.

File: data/quarg/quarg missions.txt (hunk 12 above)
Comment (Hurleveur) [reply data/quarg/quarg missions.txt:283]:
```suggestion
				`	"The Pug fancy themselves gods, but since God is dead..."`
```
Would it be better that way? Caus idk about the so....

File: data/quarg/quarg missions.txt (hunk 15 above)
Comment (Hurleveur) [reply data/quarg/quarg missions.txt:303]: Maybe they just suddenly turn back, as a way of saying this conversation is over?
All Quargs could turn around Klingon style but that'd be a bit too much, though just one could be nice

Comment (Saugia) [reply data/quarg/quarg missions.txt:303]: @Arachi-Lover Thoughts?

Comment (Arachi-Lover) [reply data/quarg/quarg missions.txt:303]: You could have the Quarg clearly signal the end of the conversation with the dismissive motion or such, and then explicitally take one more, long look at the Arfecta. The crowd can then be said to disperse in a few minutes, but a few Quarg remain, gawking at the player character and/or the Arfecta.

Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Updated this line. I think the current version addresses all feedback.

**Review by Hurleveur — APPROVED** [review]
Looks good to me now, although I'm not good for grammar and stuff, and only approving this as anyone would (its not my area)

File: data/quarg/quarg missions.txt [hunk 17]
```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
...
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
```
Comment (Pointedstick) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather around. As you go to disembark, you find a rather imposing Quarg blocking the exit while staring piercingly at your ship.`
```

File: data/quarg/quarg missions.txt [hunk 18]
```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
...
+			`	“You fancy yourself a savior or a revolutionary then?” the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
+				goto end
+			label greedy
+			`	"A baby in swaddling, as I said before."`
+				goto end
+			label end
+			`	"We give you a warning. You know not what you have child, the weapon you wield."`
```
Comment (Pointedstick) [inline data/quarg/quarg missions.txt:300]:
```suggestion
			`	"We give you a warning. You know not what you have, child - the weapon you wield."`
```

File: data/quarg/quarg missions.txt [hunk 19]
```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
...
+			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`
+			`	"And be careful to remember that we have learned to overcome it."`
```
Comment (Pointedstick) [inline data/quarg/quarg missions.txt:302]:
```suggestion
			`	"And be careful to remember that we have learned how to overcome it."`
```

File: data/quarg/quarg missions.txt [hunk 20]
```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
...
+			`	The Quarg curtly motions farewell then takes one last long look at your ship before leaving you to your business. The crowd too starts to disperse, with a few staying behind to gawk at your ship.`
```
Comment (Pointedstick) [inline data/quarg/quarg missions.txt:303]:
```suggestion
			`	The Quarg curtly motions farewell then takes one last long look at your ship before leaving you to your business. The crowd too starts to disperse, with a few staying behind to gawk at it.`
```

File: data/quarg/quarg missions.txt [hunk 21]
```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
...
+			`	As you make your way through the ring you notice all the Quarg onboard glaring at you with disgust.`
```
Comment (Pointedstick) [inline data/quarg/quarg missions.txt:304]:
```suggestion
			`	As you make your way through the ring, you notice that all the Quarg onboard glare at you with disgust.`
```

File: data/quarg/quarg missions.txt [hunk 22]
```diff
@@ -259,3 +259,47 @@ mission "Ask Quarg About Coalition Early"
...
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
+				goto end
+			label greedy
+			`	"A baby in swaddling, as I said before."`
+				goto end
+			label end
+			`	"We give you a warning. You know not what you have child, the weapon you wield."`
+			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you direct this power, and not it you."`
```
Comment (Daeridanii1) [inline data/quarg/quarg missions.txt:301]:
```suggestion
			`	"We have seen its power. We have felt its power. We have learned from its power. But you and your kind know nothing of this power and its burdens and dangers. Be careful that you are the one who directs this power, and not it you."`
```
This came across as somewhat hard to parse on my first reading, so I think that making the syntax more explicit could be beneficial.

File: data/quarg/quarg missions.txt (hunk 21 above)
Comment (Daeridanii1) [inline data/quarg/quarg missions.txt:304]:
```suggestion
			`	As you make your way through the ring you notice many of the Quarg onboard glaring at you with disgust.`
```
This feels like a stronger response than is set up by the rest of the conversation, especially considering the previous line has the stragglers of the crowd "gawking" at your ship in awe rather than regarding you with disgust. Is the crowd composed only of non-Quarg?

File: data/quarg/quarg missions.txt (hunk 17 above)
Comment (Daeridanii1) [inline data/quarg/quarg missions.txt:276]:
```suggestion
			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at you.`
```
"Piercingly" strikes me as a strange word to describe the Quarg's gaze with respect to your ship because it begs the question, "what exactly are they piercing?" (The ship's hull?) I typically think of "piercing" used to describe how one person looks at another (i.e. "staring piercingly through you"), such that they are piercing obfuscations or lies or feigned ignorance, all of which the imposing Quarg demonstrates later in the conversation.

**Review by Daeridanii1 — COMMENTED** [review]
Love this idea! I've always though it was strange that no one seems to notice that First Last is bringing their weird alien death machines into the spaceport, so it's great to see some acknowledgement of the player's actions in this way. My comments are all pretty minor semantics, but I think they could help make the story just a little bit clearer and easier on the reader.

File: data/quarg/quarg missions.txt (hunk 17 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Makes sense will update myself as this line now has a conflict.

File: data/quarg/quarg missions.txt (hunk 21 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Makes sense will update directly because of a line conflict

Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Change made

File: data/quarg/quarg missions.txt (hunk 17 above)
Comment (rovermicrover) [reply data/quarg/quarg missions.txt]: Change made

**Comment by rovermicrover** [comment]
Out and about and working off my steam deck so will load and re verify this in game tonight just in case.

**Comment by rovermicrover** [comment]
Ok verified locally

**Review by McloughlinGuy — COMMENTED** [review]
I think that having an acknowledgement for something like this is a good idea.

**Review by roadrunner56 — APPROVED** [review]

**Review by McloughlinGuy — APPROVED** [review]

File: data/quarg/quarg missions.txt [hunk 23]
```diff
@@ -274,3 +274,45 @@ mission "Quarg Ships Atrocity"
...
+					goto righteous
+				`	"The Pug fancy themselves gods, but since God is dead..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	It scoffs, "It matters not how it happened."`
+				goto end
+			label righteous
+			`	“You fancy yourself a savior or a revolutionary then?” the Quarg says with what you assume is a sarcastic tone.`
```
Comment (1010todd) [inline data/quarg/quarg missions.txt:304]:
```suggestion
			`	"You fancy yourself a savior or a revolutionary then?" the Quarg says with what you assume is a sarcastic tone.`
```
Swap fancy quote for normal one.

**Review by tibetiroka — APPROVED** [review]

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Comment by rovermicrover** [comment]
Suggestions committed and master merged

File: data/quarg/quarg missions.txt [hunk 24]
```diff
@@ -274,3 +274,51 @@ mission "Quarg Ships Atrocity"
 	on offer
 		conversation "quarg imprisonment"
 
+
+
+mission "Quarg Pug Arfecta Warning"
+	minor
+	landing
+	source
+		attributes "quarg" "ringworld"
```
Comment (Amazinite) [inline data/quarg/quarg missions.txt:283]: Multiple attributes on the same line means "A or B". In this case, we want this to offer on a planet that is both Quarg AND a ringworld, in which case we use separate attribute lines.
```suggestion
		attributes "quarg"
		attributes "ringworld"
```

**Review by Amazinite — APPROVED** [review]

File: data/quarg/quarg missions.txt [hunk 25]
```diff
@@ -274,3 +274,51 @@ mission "Quarg Ships Atrocity"
...
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	It scoffs, "It matters not how it happened.`
+				goto end
+			label righteous
+			`	"You fancy yourself a savior or a revolutionary then?" the Quarg says with what you assume is a sarcastic tone.`
+				goto end
+			label anarchy
+			`	"So you took it upon yourself to prove them wrong?" the Quarg says with an amused, incredulous smile.`
```
Comment (Ferociousfeind) [inline data/quarg/quarg missions.txt:314]: Definitely optional, but I wonder if this sounds better?

```suggestion
			`	"So you deign to prove them wrong?" the Quarg says with an almost amused, or perhaps incredulous tone.`
```

**Review by Ferociousfeind — COMMENTED** [review]
Don't need to accept any of these, but these are my own writer thoughts

File: data/quarg/quarg missions.txt [hunk 26]
```diff
@@ -274,3 +274,52 @@ mission "Quarg Ships Atrocity"
...
+		attributes "quarg"
+		attributes "ringworld"
+	to offer
+		has "First Contact: Quarg: offered"
+		has "ship model: Pug Arfecta"
+	on offer
+		conversation
+			branch flagship
+				has "flagship model: Pug Arfecta"
+			`As soon as you land, you see a crowd of Quarg gathering around one of your escorts, the Pug Arfecta. The captain of the ship looks shaken as the Quarg surround them, seemingly being questioned by a rather tall Quarg, even by Quarg standards. The captain looks around and, seeing you, points in your direction, causing the Quarg to come your way. The tall Quarg towers over you and gives you a piercing stare.`
```
Comment (Hurleveur) [inline data/quarg/quarg missions.txt:292]: What if this is your flagship?

Comment (Amazinite) [reply data/quarg/quarg missions.txt:292]: Then it goes to the label from the branch right above this line.

Comment (Hurleveur) [reply data/quarg/quarg missions.txt:292]: Oh yeah I'm blind, phone review ig
Sorry
